### PR TITLE
Create quantmod_translation.Rmd

### PR DESCRIPTION
**Pull request by jx2428**
# CC from Jiashu (Jerry) Xia

### CODE REVIEW

**Review by yosha-tomar — CHANGES_REQUESTED** [review]
Please make the following changes:
1. Completely remove the YAML header and use the following format:

    # TITLE
 
    Your Name

     Your content

For reference see the sample file: https://github.com/jtr13/cc21/blob/main/sample_project.Rmd
And the second point of section 2.2 in:
https://jtr13.github.io/cc21/github-submission-instructions.html#preparing-your-.rmd-file

2. Remove setup label from line 16.
For reference see point 6 of section 2.2 in:
https://jtr13.github.io/cc21/github-submission-instructions.html#preparing-your-.rmd-file

File: quantmod_translation.Rmd [hunk 1]
```diff
@@ -0,0 +1,155 @@
+
+R Quantmod Package Tutorial & Translation
```
Comment (yosha-tomar) [inline quantmod_translation.Rmd:2]: Change Title to "# R Quantmod Package Tutorial & Translation"

**Review by yosha-tomar — CHANGES_REQUESTED** [review]
Thanks for the changes. Please write the title in the correct format. It should begin with a single hashtag followed by a space and then your title.

**Comment by yosha-tomar** [comment]
Thank you for your contribution!

**Comment by shauryamalik** [comment]
I see the build failed for this merge: https://github.com/jtr13/cc21/runs/2241670443 @yosha-tomar

**Comment by yosha-tomar** [comment]
I'm working on it. 
The Rmd file had some characters which were creating the issue.

**Pull request merged**